src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -651,6 +651,9 @@ public SyntaxNode GetExpressionOfArgument(SyntaxNode node)
         public RefKind GetRefKindOfArgument(SyntaxNode node)
             => (node as ArgumentSyntax).GetRefKind();
 
+        public bool IsArgument(SyntaxNode node)
+            => node is ArgumentSyntax;
```

Comment:
💭 This makes me wonder why we aren't using `node.IsKind(SyntaxKind.Argument)`, but I see other methods like `IsCastExpression` doing the same thing. Should be fine.

Reply:
It never showed up in perf traces, and it kept things simple i.e. ```IsArgument -> "is Argument"```.  But feel free to change for any nodes that has a 1:1 correspondence with kinds.

Reply:
I removed this method. I don't need it anymore with the change that @CyrusNajmabadi suggested in the code fix class.